### PR TITLE
Added scripts for remaining Nov 9 mailings

### PR DESCRIPTION
**Pull request by aoby**
This is for 
1. the remaining general list (results of yesterday's A/B testing)
2. all teachers.

### CODE REVIEW

File: bin/hoc-2015-nov-9-mailing/send-general-final-real-mails [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+# ubuntu@production-console:~/production/bin/hoc-2015-nov-9-mailing$ wc -l general-deduped-remainder.csv
+# 1261412 general-deduped-remainder.csv
+# 1,261,412
+
+./send-to-mailing-list --start       0 --limit 100000 11-9-hoc-general hoc-2015-nov-9-mailing/general-deduped-test-remainder.csv
```
Comment (laurelfan) [inline bin/hoc-2015-nov-9-mailing/send-general-final-real-mails]: the filename is actually general_deduped_test_remainder?

Comment (aoby) [inline bin/hoc-2015-nov-9-mailing/send-general-final-real-mails]: nope

https://github.com/code-dot-org/code-dot-org/blob/staging/bin/mailing-common/split#L52

```
ubuntu@production-console:~/production/bin/hoc-2015-nov-9-mailing$ ls general-deduped* -l
-rw-rw-r-- 1 ubuntu ubuntu 54987500 Nov 10 02:02 general-deduped.csv
-rw-rw-r-- 1 ubuntu ubuntu 50035378 Nov 10 02:02 general-deduped-remainder.csv
-rw-rw-r-- 1 ubuntu ubuntu  1650255 Nov 10 02:02 general-deduped-test-1.csv
-rw-rw-r-- 1 ubuntu ubuntu  1651565 Nov 10 02:02 general-deduped-test-2.csv
-rw-rw-r-- 1 ubuntu ubuntu  1650377 Nov 10 02:02 general-deduped-test-3.csv
```

Comment (laurelfan) [inline bin/hoc-2015-nov-9-mailing/send-general-final-real-mails]: ok so you need to fix this line and the rest? (this line says `hoc-2015-nov-9-mailing/general-deduped-test-remainder.csv` and it looks like you want `hoc-2015-nov-9-mailing/general-deduped-remainder.csv` ?)

Comment (aoby) [inline bin/hoc-2015-nov-9-mailing/send-general-final-real-mails]: oh, I see what you're saying. That part was off my screen; I just saw the comment above. Thanks. 

Fixed.

**Comment by laurelfan** [comment]
LGTM

**Pull request merged**